plugins/ci/commands/analyze-regression.md

```
@@ -0,0 +1,174 @@
+- Integrates with existing `prow-job:analyze-test-failure` for root cause analysis
+- Helps teams consistently follow the documented triage procedure
+- For high-level component health analysis, use `/component-health:analyze-regressions` instead
+- For listing all regressions, use `/component-health:list-regressions`
+- For questions, ask in #forum-ocp-release-oversight
+
+## See Also
+
+- Related Command: `/component-health:list-regressions` (for bulk regression data)
+- Related Command: `/component-health:analyze-regressions` (for overall component health)
+- Related Command: `/prow-job:analyze-test-failure` (for test root cause analysis)
+- Component Readiness: https://sippy-auth.dptools.openshift.org/sippy-ng/component_readiness/main
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)